yappi/yappi.py

```diff
@@ -1108,13 +1230,13 @@ def get_thread_stats():
     """
     Gets the thread profiler results with given filters and returns an iterable.
     """
-    _yappi._pause()
-    try:
-        stats = YThreadStats().get()
-    finally:
-        _yappi._resume()
-    return stats
```

Comment:
Why have we removed `_pause` and `_resume`?

Reply:
pause and resume were already being invoked in the _YContextStats (YThreadStats earlier) and so I thought it isn't required here as well. Please correct me if I am missing something.

Reply:
Ah I see! You are right.